src/platforms/flutter/native-init.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Native Initialization
+sidebar_order: 900
+description: "Learn how to manually initialize the native SDKs."
+---
+
+By default, the Flutter SDK initializes the native SDK underneath the `init` method called on the Flutter layer. As a result, the SDK has a current limitation of not capturing native crashes that occur prior to the `init` method being called on the Flutter layer. You can initialize the native SDKs yourself to overcome this limitation or if you want to provide custom options above what the Flutter SDK currently provides.
+
+To do this, set [autoInitializeNativeSdk](/platforms/flutter/configuration/options/#autoInitializeNativeSdk) to `false` in the init options:
+
+```dart
+import 'package:sentry/sentry.dart';
+
+Future<void> main() async {
+  await Sentry.init((options) => options
+    ..dsn = '___PUBLIC_DSN___'
+    ..autoInitializeNativeSdk = false);
+}
+```
+
+This will prevent the Flutter SDK from initializing the native SDK automatically.
+
+Next, initialize the native SDKs for Android, which is documented below, or for iOS by following the [configuration guide to initialize the Sentry Cocoa SDK](/platforms/apple/guides/ios/#configure). Note that you do not need to install the native SDKs as they are already packaged with the Flutter SDK.
+
+## Android Configuration
+
+To use auto-init, add the following to your `AndroidManifest.xml`:
+
+```xml
+<meta-data android:name="io.sentry.auto-init" tools:replace="android:value" android:value="true" />
+```
```

Comment:
I believe this won't work because the Flutter SDK removes the Content provider https://github.com/getsentry/sentry-dart/blob/main/flutter/android/src/main/AndroidManifest.xml#L5-L9=
Users have to manually init. the SDK so I'd rather just link it to https://docs.sentry.io/platforms/android/configuration/manual-init/